src/notifier.c

```diff
@@ -343,6 +343,10 @@ notification_show(IndicatorPowerNotifier * self)
             }
         }
 
+      notify_notification_set_hint(nn, "x-lomiri-snap-decisions", g_variant_new_string("true"));
+      notify_notification_set_hint(nn, "x-lomiri-non-shaped-icon", g_variant_new_string("true"));
+      notify_notification_set_hint(nn, "x-lomiri-private-affirmative-tint", g_variant_new_string("true"));
+      notify_notification_set_hint(nn, "x-lomiri-snap-decisions-timeout", g_variant_new_int32(INT32_MAX));
```

Comment:
This should be encapsulated in

```
#ifdef LOMIRI_FEATURES_ENABLED
...
#endif
```

and

```
if (ayatana_common_utils_is_lomiri()) {
    ...
}
```

Reply:
I've decided to use notification server capability check instead of using utility function. Although, it turns out to be inconsistent with the rest of the code base. What do you think?